cc-pk-service/src/app.ts

```diff
@@ -0,0 +1,144 @@
+import express from 'express';
+import axios from 'axios';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+// Variables
+const PORT = process.env.PORT;
+const username: string = process.env.username ?? '';
+const password: string = process.env.password ?? '';
+const c_id: string = process.env.clientid ?? '';
+const g_type: string = process.env.granttype ?? '';
+const path = '/' + username;

```

Comment:
when creating strings variables with str and variables, would use template literals, like 

```suggestion
const path =`/${username}`;
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals